src/js/providers/video-listener-mixin.js

```diff
@@ -48,7 +48,7 @@ const VideoListenerMixin = {
 
         if (!this.seeking && !this.video.paused &&
             (this.state === STATE_STALLED || this.state === STATE_LOADING) &&
-            this.stallTime !== currentTime) {
+            this.stallTime !== this.video.currentTime) {
```

Comment:
Why did you make this change when `currentTime` was assigned the value of `this.video.currentTime` above?

Reply:
@robwalch `currentTime` can change between allocation and usage. This is the case with the current bug; shaka seeks backwards after. But I can't remember if it was the case that `this.video.currentTime` was equal to `this.stallTime`, and that `currentTime` wasn't; @DanFerrer Can you double check if this is still necessary?